js/ui/diagram/ui.diagram.js

```diff
@@ -672,6 +675,14 @@ class Diagram extends Widget {
             this._nodesOption.remove(key, callback);
         }
     }
+    _raiseToolboxDragStart() {
+        const body = getWindow().document.body;
```

Comment:
I'd check the server mode via the `hasWindow()` method because a diagram's instance is initialized in server mode as well.

Reply:
Do we really need document.body? Can we add the class on the toolbox container (this._leftPanel.$element())?